src/frontend/js/delivery.js

```diff
@@ -7,4 +7,12 @@ $(function() {
         window.location.replace($url+value);
     });
 
+    $('.button.orders').click(function(){
+        $('.button.orders i').addClass('loading');
+        var url = $(this).attr('data-url');
+        $.get(url, function( data ) {
+            window.location.reload();
```

Comment:
@manumilou `data` should contain the list of orders generated something like:
```
data = { 
   {
      id : "978",
      client_id: "611",
      client_full_name: "Kevin Tracy",
      date: "Nov. 28, 2016",
      route: "Mile-End",
      status: "Ordered",
      price: "9.00",
      url: "/order/view/978/"
   },
   {...}
}
```
You could use jQuery to iterate over each `oder` in `data` and add it to the `tbody` element in tha page.. ?